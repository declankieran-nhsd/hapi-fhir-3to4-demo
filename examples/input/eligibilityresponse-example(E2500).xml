<?xml version="1.0" encoding="UTF-8"?><EligibilityResponse xmlns="http://hl7.org/fhir">

	<text>
		<status value="generated"/>
		<div xmlns="http://www.w3.org/1999/xhtml">A human-readable rendering of the EligibilityResponse.</div>
	</text>

	<identifier>
		<system value="http://www.BenefitsInc.com/fhir/eligibilityresponse"/>
		<value value="881234"/>
	</identifier>

	<status value="active"/>

	<created value="2014-08-16"/>

	<request>
		<reference value="http://www.BenefitsInc.com/fhir/eligibility/225476332402"/>
	</request>

	<outcome>
		<coding>
			<system value="http://hl7.org/fhir/remittance-outcome"/>
			<code value="complete"/>
		</coding>
	</outcome>

	<disposition value="Policy is currently in-force."/>

	<insurer>
		<reference value="Organization/2"/>
	</insurer>

	<inforce value="true"/>

</EligibilityResponse>