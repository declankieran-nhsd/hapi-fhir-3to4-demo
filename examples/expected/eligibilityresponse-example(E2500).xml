<?xml version="1.0" encoding="UTF-8"?>

<CoverageEligibilityResponse xmlns="http://hl7.org/fhir">
  <text>
    <status value="generated"/><div xmlns="http://www.w3.org/1999/xhtml">A human-readable rendering of the EligibilityResponse.</div>
  </text>
  <identifier>
    <system value="http://www.BenefitsInc.com/fhir/eligibilityresponse"/>
    <value value="881234"/>
  </identifier>
  <status value="active"/>
  <created value="2014-08-16"/>
  <request>
    <reference value="http://www.BenefitsInc.com/fhir/eligibility/225476332402"/>
  </request>
  <outcome value="complete"/>
  <disposition value="Policy is currently in-force."/>
  <insurer>
    <reference value="Organization/2"/>
  </insurer>
</CoverageEligibilityResponse>